qemu-run/Cargo.toml

```diff
@@ -8,5 +8,11 @@ version = "0.1.0"
 
 [dependencies]
 anyhow = "1.0.32"
-defmt-decoder = { path = "../decoder" }
+defmt-decoder = { path = "../decoder", features = ["unstable"] }
 defmt-elf2table = { path = "../elf2table" }
+
+[features]
+unstable = []
+
+[package.metadata.docs.rs]
+rustdoc-args = [ "--cfg docsrs" ]
```

Comment:
This package is marked `publish = false`, so it won't end up on crates.io, which means we don't have to bother with stability (unless we *do* decide that we want to publish it, but I don't really think we will – this program is just meant to power defmt's test suite, nothing else)

Reply:
wait, if that's possible why don't we just do that for e.g. `elf2table` too?